packages/beacon-node/test/unit/execution/engine/utils.test.ts

```diff
@@ -0,0 +1,191 @@
+import {expect} from "chai";
+import {ErrorAborted} from "@lodestar/utils";
+import {ExecutePayloadStatus, ExecutionEngineState} from "../../../../src/execution/index.js";
+import {
+  HTTP_CONNECTION_ERROR_CODES,
+  HTTP_FATAL_ERROR_CODES,
+  getExecutionEngineState,
+} from "../../../../src/execution/engine/utils.js";
+import {QueueError, QueueErrorCode} from "../../../../src/util/queue/errors.js";
+import {ErrorJsonRpcResponse, HttpRpcError} from "../../../../src/eth1/provider/jsonRpcHttpClient.js";
+
+describe("execution/engine/utils", () => {
```

Comment:
to be more consistent with existing test titles

```suggestion
describe("execution / engine / utils", () => {
```